src/lib/y2partitioner/actions/controllers/encryption.rb

```diff
@@ -186,6 +182,9 @@ def can_change_encrypt?
           filesystem.nil? || new?(filesystem)
         end
 
+        # Returns the filesystem
+        #
+        # @return [Y2Storage::Filesystem, nil] block device filesystem, if any
```

Comment:
Please, re-word: `# @return [Y2Storage::Filesystem, nil] nil if the plain block device is not formatted`